utils/errors/errors.go

```diff
@@ -37,13 +37,19 @@ type NotifyExt interface {
 	Notified(status bool)
 }
 
+// RawExt is the interface definition for raw data about an error
```

Comment:
Is it necessary to create a new interface just  to implement this method or maybe it could be declared in the original ErrorExt interface? In my opinion, the more interface declared, the more effort that developers need to do to customize their own error struct on top of Orion error.

Reply:
I don't think more interfaces will make relative efforts to customizing on your own errors, thus it should make things easier. Ideally, you can embed interfaces you need rather than you are forced to implement all of functions if we put all functions in only one interface here.